installing/installing_azure/preparing-to-install-on-azure.adoc

```diff
@@ -0,0 +1,44 @@
+[id="preparing-to-install-on-azure"]
+= Preparing to install on Azure
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-azure
+
+toc::[]
+
+[id="preparing-to-install-on-azure-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+[id="requirements-for-installing-ocp-on-azure"]
+== Requirements for installing {product-title} on Azure
+
+Before installing {product-title} on Microsoft Azure, you must configure an Azure account. See xref:../../installing/installing_azure/installing-azure-account.adoc#installing-azure-account[Configuring an Azure account] for details about account configuration, account limits, public DNS zone configuration, required roles, creating service principals, and supported Azure regions.
+
+If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, see xref:../../installing/installing_azure/manually-creating-iam-azure.adoc#manually-creating-iam-azure[Manually creating IAM for Azure] for other options.
+
+[id="choosing-an-method-to-install-ocp-on-azure"]
+== Choosing a method to install {product-title} on Azure
+
+You can install {product-title} on Azure by using one of the following deployment methods.
+
+* **xref:../../installing/installing_azure/installing-azure-default.adoc#installing-azure-default[Installing a cluster quickly on Azure]**: You can install {product-title} on Azure by using the default configuration options.
```

Comment:
Does the installation program provision the infrastructure?

Reply:
In this case yes. Thank you for highlighting the need for clarification. I have updated the bullet point as follows:

`You can install {product-title} on Azure infrastructure that is provisioned by the {product-title} installation program. You can install a cluster quickly by using the default configuration options.`

I have also updated the equivalent line in the existing "Preparing to install on AWS" section.